sys-process/rtirq/rtirq-20180209.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit systemd
+
+DESCRIPTION="Modify realtime scheduling policy and priority of IRQ handlers"
+HOMEPAGE="http://www.rncbc.org/archive/#rtirq"
+
+SRC_URI="http://www.rncbc.org/archive/${P}.tar.gz"
+LICENSE="GPL-2"
```

Comment:
Could you verify whether this isn't `GPL-2+` actually? Look at the copyright notice on top of source files; if it says 'GNU General Public License, version 2 or (at your blah blah) newer version', then it should be `GPL-2+`.

Reply:
Okay, the `rtirq.sh` script says:

```
#   This program is free software; you can redistribute it and/or
#   modify it under the terms of the GNU General Public License
#   as published by the Free Software Foundation; either version 2
#   of the License, or (at your option) any later version.
```

so the `LICENSE` variable has been updated accordingly.